contracts/near/eth-prover/src/tests_unlock.rs

```diff
@@ -0,0 +1,296 @@
+#[cfg(test)]
+mod tests_unlock {
```

Comment:
-> `tests_storage_proof`?

Reply:
This will require file change too, should I do it?

Reply:
Sure, the name of the files should also be changed.

Reply:
changed in 7b92a6305061374756668800d12a1cab623da9aa